docs/docs/tutorial/chapter4/authentication.md

```diff
@@ -474,7 +474,7 @@ Yet another generator is here for you, this time one that will create pages for
 yarn rw g dbAuth
 ```
 
-Again several pages will be created and some post-install instructions will describe next steps. But for now, try going to [http://localhost:8910/login](http://localhost:8910/login):
+Say no when asked to Enable WebAuthn support. Again several pages will be created and some post-install instructions will describe next steps. But for now, try going to [http://localhost:8910/login](http://localhost:8910/login):
```

Comment:
No need to repeat this here again, I think!

Reply:
Sure, deleted.